superset-frontend/src/components/DropdownSelectableIcon/index.tsx

```diff
@@ -98,7 +104,19 @@ const StyleSubmenuItem = styled.div`
 
 export default (props: DropDownSelectableProps) => {
   const theme = useTheme();
+  const [visible, setVisible] = useState(false);
   const { icon, info, menuItems, selectedKeys, onSelect } = props;
+
+  const handleVisibleChange = (flag: boolean) => {
+    setVisible(flag);
+  };
```

Comment:
```suggestion
  const handleVisibleChange = setVisible;
```

Or just use `setVisible` where `handleVisibleChange` is being used.